src/utils/daos.c

```diff
@@ -1091,6 +1108,12 @@ cont_op_hdlr(struct cmd_args_s *ap)
 	case CONT_DESTROY:
 		rc = cont_destroy_hdlr(ap);
 		break;
+	case CONT_COPY:
+		rc = cont_copy_hdlr(ap);
+		break;
+
+
+	/* TODO: implement the following ops */
```

Comment:
@dsikich Question:
Should this TODO have been added? Isn't this implemented?

Reply:
@daltonbohning I don't know that was already there. I did not add that TODO.

Reply:
@dsikich I **think** that TODO was there before you started this branch, but then it was resolved _**during**_ your development of this branch. Maybe a merge accidentally put it back in? I do believe `daos obj list` works now, and in _**master**_ this TODO is not there

Reply:
For reference, in master:
https://github.com/daos-stack/daos/blob/130088667ac84bde8437df97027e13518306adf9/src/utils/daos.c#L1091-L1097

Reply:
@daltonbohning ok, I removed it. It must have stayed there because I added the CONT_COPY.